packages/containers/Import/Steps/DataStep/DataStep.js

```diff
@@ -346,16 +412,20 @@ class DataStep extends Component {
 
     const selectedRowsLength = denormalize(selectedRows).length;
 
-    const teamOptions = teamsLists.map(({ id, name }) => ({
-      value: id,
-      label: name.toLowerCase(),
-    }));
+    const teamOptions = teamsLists
+      .filter(({ id }) => id === TEAM_TYPE.PERSONAL)
+      .map(({ id, name }) => ({
+        value: id,
+        label: name.toLowerCase(),
+      }));
```

Comment:
For Fun, you can use `flatMap` :smile:  I found out about it recently) In your case:
```
const teamOptions = teamsLists
  .flatMap({ id, name } => id !== TEAM_TYPE.PERSONAL 
  ? [] 
  : ({ 
    value: id, 
    label: name.toLowerCase()  
  }));
```
More information https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap#For_adding_and_removing_items_during_a_map

I think, it might be interesting :smiley_cat:

Reply:
oh, thank u!
sound good